integration-tests/test-queries/more-meta.js

```diff
@@ -0,0 +1,42 @@
+export const query = `
+query foo {
```

Comment:
Tiny nitpick, doesn't block anything, but would be good to have a descriptive name here so anyone can instantly tell what this for

Reply:
the filename says it all imo